### ISSUE TITLE
rows with identical observations are removed during normalization

### ISSUE THREAD

**Issue by cells2numbers** [shntnu/pertminr#53]
In the example below a population with 8 rows is normalized. The normalized version of the data includes only 7 rows: all rows with identical entries are summarized in one row. 

This error will probably not happen during the analysis of profiling data; however, in the simple test data it shows up. 

```
variables <- c("AreaShape_Area")
strata <- c("Metadata_batch")
sample <- population %>% filter(Metadata_group == "control")

population <- tibble::data_frame(
   Metadata_group = c("control", "control","control","control","experiment","experiment","experiment","experiment"),
   Metadata_batch = c("a","a","b","b","a","a","b","b"),
   AreaShape_Area = c(10,12,15,16,8,9,7,7)
 ) %>%
  print

normalized = cytominer::normalize(population, variables, strata, sample, operation = "standardize") %>% print
```

Population:
```
Metadata_group<chr> | Metadata_batch<chr> | AreaShape_Area<dbl> |   |  
-- | -- | -- | -- | --
control | a | 10 |   |  
control | a | 12 |   |  
control | b | 15 |   |  
control | b | 16 |   |  
experiment | a | 8 |   |  
experiment | a | 9 |   |  
experiment | b | 7 |   |  
experiment | b | 7 |   |  
```
normalized
```
Metadata_group<chr> | Metadata_batch<chr> | AreaShape_Area<dbl> |   |  
-- | -- | -- | -- | --
experiment | b | -12.0208153 |   |  
control | b | 0.7071068 |   |  
control | b | -0.7071068 |   |  
experiment | a | -1.4142136 |   |  
experiment | a | -2.1213203 |   |  
control | a | 0.7071068 |   |  
control | a | -0.7071068 |   |  
```

**Comment by shntnu** [comment]
Yes this is because of using union instead of union_all. This is fixed in
the branch I'm working on

**Comment by shntnu** [comment]
Fixed in #54
